console/console-init/ui/src/modules/iot-device/containers/DeviceListContainer/DeviceListContainer.tsx

```diff
@@ -110,14 +110,19 @@ export const DeviceListContainer: React.FC<IDeviceListContainerProps> = ({
       title: "Delete",
       onClick: () => deleteDevice(rowData)
     },
-    {
-      title: "Disable",
-      onClick: () => disableDevice(rowData)
-    },
-    {
-      title: "Enable",
-      onClick: () => enableDevice(rowData)
-    }
+    ...(rowData.originalData.enabled
```

Comment:
you can make it  little bit short.
const enabled=rowData?.originalData?.enabled;
...( 
[
          {
            title: enabled ? "Disable" "Enable",
            onClick: () => enabled ?  disableDevice(rowData) :enableDevice(rowData)
          }
        ]
)

Reply:
Done